libcodechecker/libhandlers/analyze.py

```diff
@@ -470,6 +471,37 @@ def __init__(self, args=None):
                 self.compiler_target_file = args.compiler_info_file
 
 
+def __get_skip_handler(args):
+    """
+    Initialize and return a skiplist handler if
+    there is a skip list file in the arguments.
+    """
+    try:
+        if args.skipfile:
+            LOG.debug_analyzer("Creating skiplist handler.")
+            with open(args.skipfile) as skip_file:
+                return skiplist_handler.SkipListHandler(skip_file.read())
+    except AttributeError:
+        LOG.debug_analyzer('Skip file was not set in the command line')
+
+    return None
+
+
+def __remove_skipfile(args):
```

Comment:
Maybe it would be a nice addition to the refactoring to rename this function to something like `replace_skip_file` or `update_skip_file` at the output directory.
What we do here is we copy the user's skip file to the output directory so it will be sent to the storage server when the results are stored.

Reply:
Done.